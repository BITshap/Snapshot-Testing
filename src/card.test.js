import Card from './card';
import renderer from 'react-test-renderer'


test('renders a snapshot', () => {
  const tree = renderer.create(<Card/>).toJSON()
  expect(tree).toMatchSnapshot()
})