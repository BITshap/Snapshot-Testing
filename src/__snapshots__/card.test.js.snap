// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
  style={Object {}}
>
  <img
    className="card-img-top"
    src=""
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    />
    <p
      className="card-text"
    >
      Hello! This is Nick Shapoff, a software developer making cool things for others!
    </p>
  </div>
</div>
`;
